src/components/common/SubItem/SubItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useCallback } from "react";
+import { P5, Spinner, Checkbox } from "@deskpro/deskpro-ui";
+import { Card } from "../Card";
+import type { FC } from "react";
+import type { ITaskFromList } from "@/services/wrike/types";
+
+export type Props = {
+  item: ITaskFromList;
+  onComplete?: (
+    itemId: ITaskFromList["id"],
+    complete: boolean,
+  ) => Promise<unknown>;
+};
+
+const SubItem: FC<Props> = ({ item, onComplete }) => {
+  const boxSize = 14;
```

Comment:
Is this because there is nothing in `@deskpro/ui`?

Reply:
it's to make it look nice. By default, in `deskpro-ui` it’s 12px

<img width="270" alt="Screenshot 2024-09-18 at 17 46 02" src="https://github.com/user-attachments/assets/b51dd511-4198-4846-95a2-4d8cf51bfd22">